test_scripts/API/AddCommand/005_ATF_AddCommand_UI_AddCommand_WARNINGS.lua

```diff
@@ -0,0 +1,142 @@
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.. "/"
+local ServerAddress = commonFunctions:read_parameter_from_smart_device_link_ini("ServerAddress") 
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"AddCommand")
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)   
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+local resultCodes = {"SUCCESS", "WARNINGS", "WRONG_LANGUAGE", "RETRY", "SAVED"}
+  for i=1,#resultCodes do
+    Test["TestStep_AddCommand_UI_AddCmd_WARNINGS_and_VR_AddCmd_"..resultCodes[i]] = function(self)
+    local cid = self.mobileSession:SendRPC("AddCommand",
+    {
+      cmdID = 11,
+      menuParams =
+      {
+        parentID = 0,
+        position = 0,
+        menuName ="Commandpositive"
+      },
+      vrCommands =
+      { 
+        "VRCommandonepositive",
+        "VRCommandonepositivedouble"
+      },
+      cmdIcon =
+      { 
+        value ="icon.png",
+        imageType ="DYNAMIC"
+      }
+    })
+    EXPECT_HMICALL("UI.AddCommand",
+    {
+      cmdID = 11,
+      cmdIcon =
+      {
+        value = storagePath .."icon.png",
```

Comment:
@vvvakulenko, this may fail when SDL is using absolute path to icon.
Verification can be done with
:ValidIf(function(_,data)

As example see, your script 003_ATF_DeleteCommand_UI_DeleteCommand_WARNINGS.lua